_pages/home.md

```diff
@@ -17,33 +17,34 @@ intro:
   - excerpt: "
 <div class='excerpt-title'>Latest News</div>
 
+<style> li{width:100%} </style>
 <ul>
   <li>
-    <a href='/release/pack-downloads'>Apache ServiceComb Pack 0.5.0 is released</a>
+      <a href='/release/'>Apache ServiceComb Release Version 1.2.0</a>
   </li>
   <li>
-    <a href='/release/java-chassis'>Apache ServiceComb Java-Chassis Release Version 1.2.1</a>
+    <a href='/release/pack-downloads'>Apache ServiceComb Pack 0.5.0 is released</a>
   </li>
   <li>
-    <a href='/docs/mersher-quick-start/'>Apache ServiceComb new project Mesher </a>
+    <a href='/release/toolkit-downloads'>Apache ServiceComb Toolkit 0.1.0 is released</a>
   </li>
   <li>
-    <a href=''>Apache ServiceComb won the first prize of China's outstanding open source project organized by China Open Source Cloud Alliance</a>
+    <a href='/release/java-chassis'>Apache ServiceComb Java-Chassis Release Version 1.2.1</a>
```

Comment:
path of '/release/java-chassis' is incorrect

Reply:
thanks, I have fixed this problem in the new commit

Reply:
thanks, I have fixed this problem in the new commit, pls help to check